ci/pipeline.yaml

```diff
@@ -109,16 +116,38 @@ jobs:
             tag: version/version
             globs:
               - build/*
-        - put: docker-image
-          inputs:
-            - build
-            - source
-            - version
-          params:
-            build: .
-            dockerfile: source/Dockerfile
-            tag_as_latest: false
-            tag_file: version/version
+        - do:
+          - file: version/version
+            load_var: image_version
+          - get: ubuntu-image
+            params:
+              format: oci
+          - get: oci-build-task
+          - task: build-oci
+            privileged: true
+            image: oci-build-task
+            config:
+              platform: linux
+              inputs:
+                - name: source
+                - name: build
+                - name: ubuntu-image
+              outputs:
+                - name: image
+              params:
+                CONTEXT: .
+                DOCKERFILE: source/Dockerfile
+                IMAGE_ARG_ubuntu_image: ubuntu-image/image.tar
+              run:
+                path: build
```

Comment:
Similar to needs, not sure if there is a path conflict here or if the prior build task needs a refactor